File: app/src/main/java/com/stevensekler/android/movies/Model/UriArray.java
package com.stevensekler.android.movies.Model;

import android.net.Uri;

import java.util.ArrayList;

/**
 * Created by Szekely Istvan on 5/24/2017.
 *
 */

public class UriArray {
    private ArrayList<Uri> uriArrayList;

    public UriArray() {
        this.uriArrayList = new ArrayList<>();
    }

    public void addToUriArray(Uri uri) {
        uriArrayList.add(uri);
    }

    public int uriArraySize() {
        return uriArrayList.size();
    }

    public Uri getUriArray(int i) {
        return uriArrayList.get(i);
    }
}

File: app/src/main/java/com/stevensekler/android/movies/Model/UriArray.kt
package com.stevensekler.android.movies.Model

import android.net.Uri

import java.util.ArrayList

/**
 * Created by Szekely Istvan on 5/24/2017.
 *
 */

class UriArray {
    private val uriArrayList: ArrayList<Uri> = ArrayList()

    fun addToUriArray(uri: Uri) {
        uriArrayList.add(uri)
    }

    fun uriArraySize(): Int {
        return uriArrayList.size
    }

    fun getUriArray(i: Int): Uri {
        return uriArrayList[i]
    }
}
